packages/scope-manager/src/lib/index.ts

```diff
@@ -216,5 +216,3 @@ const lib = {
   'webworker.importscripts': webworker_importscripts,
   'webworker.iterable': webworker_iterable,
 } as const;
-
-export { lib };
```

Comment:
Sir this file is generated.

Reply:
Yes! And it shall now be generated in a way pleasing to me: https://github.com/typescript-eslint/typescript-eslint/pull/10596/files#diff-f7f9f02321a83a9f3641e294a36f1513f61b7ac40a11751a29ef193f36a783f1